Formalist/AccessoryViewToolbar.swift

```diff
@@ -21,19 +21,24 @@ class AccessoryViewToolbar: UIToolbar {
         return UIBarButtonItem(title: "Next", style: .plain, target: self, action: #selector(nextAction))
     }()
 
-    lazy var doneButtonItem: UIBarButtonItem = {
-        return UIBarButtonItem(barButtonSystemItem: .done, target: self, action: #selector(done))
-    }()
+    private func doneButtonItem(customTitled title:String? = nil) -> UIBarButtonItem {
```

Comment:
Did you consider `doneButton(withTitle:)`?

Reply:
That does read better – I don't remember why I decided not to use 'with', but I'm on board

Reply:
Oh, I think part of it was wanting to communicate that `customTitle: nil` would fall back to a default title, where a `title: nil` sounds like it might have no title at all. 

`withCustomTitle` maybe?

I'd really love to keep taking advantage of the system done button fanciness – they did the work to translate 'done' into a short, sweet, standard word in every language, and this way we just get that for free.

Reply:
`withCustomTitle` works for me.